index.html

```diff
@@ -240,7 +240,7 @@ <h3>Create</h3>
 {
   "@context": "https://w3id.org/did/v1",
```

Comment:
@acarnagey actually canyou fix this as well

Reply:
use https://did.key.transmute.industries/ and `application/did+ld+json`... consider adding data for the `did+json` and JWK format as well.